guides/v2.1/security/google-recaptcha.md

```diff
@@ -0,0 +1,47 @@
+---
+group: security
+title: Google reCAPTCHA
+functional_areas:
+  - Configuration
+---
+
+This extension adds the Google reCAPTCHA module to your Magento installation. Google reCAPTCHA provides a greater level of security for both the storefront and Admin than what is available with the standard CAPTCHA, and gives you the ability to:
+
+- Verify when customers create accounts, retrieve passwords, log in to their accounts, or contact your company.
+- Enhance security when Admin users log in.
+
+Google reCAPTCHA reduces potential user error when entering a series of letters and numbers, and encourages cart conversion without adding hurdles during checkout.
+
+At this time, Google reCAPTCHA can be installed only from the command line.
+
+{:.bs-callout .bs-callout-info}
+**Magento Community Contribution** - Magento thanks [Riccardo Tempesta](https://twitter.com/rictempesta) of [MageSpecialist](https://partners.magento.com/portal/details/partner/id/129) for contributing these features as part of the Magento Community Engineering program.
+
+## Install Module
+
+You will need to install the extension using composer for v2.1 and v2.2. The features will be included in default installations and upgrades of v2.3.
```

Comment:
You must install the extension using composer ....

The Google reCAPTCHA features are included by default when you install or upgrade to v2.3. 

(Stylistic: avoiding future tense)

Reply:
Revised further to "Install the extension...   and "Installations and upgrades of v2.3 include 2FA features." Also revised the intro sentence for the composer command, caps for Composer.